cmd/main.go

```diff
@@ -19,10 +22,10 @@ import (
 )
 
 const (
-	get    = "GET"
-	put    = "PUT"
-	post   = "POST"
-	delete = "DELETE"
+	codeGet    = "GET"
+	codePut    = "PUT"
+	codePost   = "POST"
+	codeDelete = "DELETE"
```

Comment:
There's no need for this. You can use existing codes from the lib like:
```go
func parseCode(code string) (codes.Code, error) {
	ret, err := codes.ToCode(code)
	switch ret {
	case codes.GET, codes.PUT, codes.POST, codes.DELETE:
		return ret, nil
	default:
		return 0, errors.Join(errCodeNotSupported, err)
	}
}
```
where `errCodeNotSupported` is:
```go
var errCodeNotSupported = errors.New("message can be GET, POST, PUT or DELETE")
```